bftengine/src/bftengine/KeyManager.cpp

```diff
@@ -322,7 +322,7 @@ bool KeyManager::KeysView::load() {
   }
   auto str = secStore->load();
   if (str.empty()) {
-    LOG_ERROR(KEY_EX_LOG, "Got empty string from loader");
+    LOG_WARN(KEY_EX_LOG, "Got empty string from loader. This is expected on startup");
```

Comment:
This message isn't exactly right. If it was always expected on startup, we wouldn't need a warning :). It's expected on "first startup with an empty database."

Reply:
Yeah :), this is only expected if its a first startup with an empty database. Its better make the message elaborate, added that in the warning.